extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSFilter.java

```diff
@@ -48,7 +47,7 @@ private void processRequestedHeaders(HttpServerResponse response, String allowHe
     }
 
     private void processMethods(HttpServerResponse response, String allowMethodsValue) {
-        if (corsConfig.methods.isEmpty()) {
+        if (containsOnlyWildcard(corsConfig.headers)) {
```

Comment:
I don't understand this change?

Reply:
This change is done so that we also support * for methods as well. See new diff.